### PR TITLE
Validator owner address handling

### PR DESCRIPTION
**Pull request by olegshmuelov**
The purpose of this PR is to handle already exist OwnerAddress field.
in the next ssv-node version`CleanRegistryData` should be enabled by default
https://bloxxx.atlassian.net/browse/BLOXSSV-681

### CODE REVIEW

**Comment by amir-blox** [comment]
you can change the given contract event for tests in `./eth1/abi_parser_test.go`

File: validator/storage/share.go [hunk 1]
```diff
@@ -25,18 +25,20 @@ func (keys PubKeys) Aggregate() bls.PublicKey {
...
 type Share struct {
-	NodeID    uint64
-	PublicKey *bls.PublicKey
-	Committee map[uint64]*proto.Node
-	Metadata  *beacon.ValidatorMetadata // pointer in order to support nil
+	NodeID       uint64
+	PublicKey    *bls.PublicKey
+	Committee    map[uint64]*proto.Node
+	Metadata     *beacon.ValidatorMetadata // pointer in order to support nil
+	OwnerAddress string
```
Comment (nivBlox) [inline validator/storage/share.go:32]: what will happened for old struct that saved in storage without the owner addr?

Comment (olegshmuelov) [reply validator/storage/share.go:32]: we are going to run the node with flag `clear registry data`  by default enabled

**Review by amir-blox — APPROVED** [review]

**Comment by olegshmuelov** [comment]
The PR was hard reverted from main branch. Closing it

**Pull request merged**